compose/Makefile

```diff
@@ -164,6 +164,33 @@ test-at-check: test-at-build  ## AMAUAT: test browsers.
 	$(call compose_amauat, \
 		run --rm --no-deps archivematica-acceptance-tests /home/archivematica/acceptance-tests/simplebrowsertest.py)
 
+define AT_HELP
+
+   Archivematica acceptance tests (Listing).
+
+   The most effective way to run these tests is to run them by tag. For
+   example:
+
+      $ make test-at-behave TAGS=black-box BROWSER=Firefox
```

Comment:
I wonder why you chose this example instead of one with a tag where BROWSER is relevant.
A second example showing `test-at-black-box` would be nice.